packages/gestalt/src/TypeaheadInputField.css

```diff
@@ -27,4 +48,12 @@ html:not([dir="rtl"]) .icon {
 
 html[dir="rtl"] .icon {
   left: 0;
-}
\ No newline at end of file
+}
+
+html:not([dir="rtl"]) .inputWrapper,
+html[dir="rtl"] .inputWrapper {
+  /* Increase specificity for this class to override SearchField defaults */
```

Comment:
This styling in `SearchField.css`:
```
html:not([dir="rtl"]) .inputActive {
  padding: 8px 32px 8px 16px;
}

html[dir="rtl"] .inputActive {
  padding: 8px 16px 8px 32px;
}
 ```
 Because this has a higher [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) than just `.inputWrapper`, I had to add the `html` selector to actually be able to override it

Reply:
ahh, so it's just the bottom padding that needs to be overridden? In that case, could we separate this into two sections of CSS, one with just `.inputWrapper` specifying the display and flex-wrap, and then another with the higher specificity for the padding bottom?

Reply:
yeah, we can do that